bedrock/firefox/templates/firefox/personal.html

```diff
@@ -0,0 +1,67 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+ {% extends "base-resp.html" %}
+
+{% block page_title_prefix %}{{_('Firefox Web Browser')}} - {% endblock %}
+{% block page_title %}{{_('When it&rsquo;s personal, choose Firefox')}}{% endblock %}
+
+{% block page_image %}{{ static('img/firefox/personal/logo.png') }}{% endblock %}
+{% block page_desc %}{{_('Choose Firefox when you want to browse the Web without the Web browsing you.')}}'{% endblock %}
+
+{% block site_css %}
+  {% stylesheet 'personal' %}
+{% endblock %}
+
+{% block site_header_nav %}{% endblock %}
+{% block site_header_logo %}{% endblock %}
+
+{% block alt_header %}
+<div class="header-content">
+  <h1>{{_('When it&rsquo;s personal,<span>choose Firefox</span>')}}</h1>
+  {{ download_firefox(simple=true) }}
+
+  <div class="mobile-download">
+    <a href="https://play.google.com/store/apps/details?id=org.mozilla.firefox&referrer=utm_source%3Dmozilla%26utm_medium%3DReferral%26utm_campaign%3Dmozilla-org">
```

Comment:
Actually we have a canonical link in setting you can use here: `{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}`